### PR TITLE
add op docs

### PR DESCRIPTION
**Pull request by njzjz**
Also add op docs to ProdEnvMatA Op to show an example of how to write Op docs.

### CODE REVIEW

File: source/op/prod_env_mat_multi_device.cc [hunk 1]
```diff
@@ -10,7 +10,7 @@ REGISTER_OP("ProdEnvMatA")
     .Attr("T: {float, double} = DT_DOUBLE")
     .Input("coord: T")          //atomic coordinates
     .Input("type: int32")       //atomic type
-    .Input("natoms: int32")     //local atomic number; each type atomic number; daizheyingxiangqude atomic numbers
```
Comment (njzjz) [inline source/op/prod_env_mat_multi_device.cc:13]: @denghuilu could you clarify what is "daizheyingxiangqude"? Is it 带着影像去的?

**Comment by njzjz** [comment]
Preview at https://deepmodeling--1804.org.readthedocs.build/projects/deepmd/en/1804/api_op.html

File: source/op/prod_env_mat_multi_device.cc (hunk 1 above)
Comment (denghuilu) [reply source/op/prod_env_mat_multi_device.cc:13]: It's a typo...

Comment (njzjz) [reply source/op/prod_env_mat_multi_device.cc:13]: 😂

**Review by wanghan-iapcm — APPROVED** [review]

**Pull request merged**